### DISCUSSION TITLE
如何改变暗黑模式的主题颜色

### DISCUSSION THREAD

**Discussion by yaojing-pan** [ant-design/ant-design#40637] [Q&A]
Menu组建可以设置暗黑模式，想请教一下如何自定义暗黑模式下的默认主题

`<Menu
          theme="dark"
          mode="inline"
          defaultSelectedKeys={['1']}
          items={[
            {
              key: '1',
              icon: <UserOutlined />,
              label: 'nav 1',
            },
            {
              key: '2',
              icon: <VideoCameraOutlined />,
              label: 'nav 2',
            },
            {
              key: '3',
              icon: <UploadOutlined />,
              label: 'nav 3',
            },
          ]}
        />`

**Comment by fnoopv** [comment]
+1，同问
